npu/broadcom/BCM56850/saivs/Dockerfile

```diff
@@ -28,7 +28,7 @@ RUN git clone https://github.com/sonic-net/sonic-sairedis.git \
         && sed -i "s/^SUBDIRS = py2 py3/SUBDIRS = py3/" pyext/Makefile.am \
         && ./autogen.sh \
         && export DEB_BUILD_PROFILES=nopython2 \
-        && dpkg-buildpackage -us -uc -b --target=binary-syncd-vs --jobs=auto \
+        && dpkg-buildpackage -us -uc -b -Psyncd,vs,nopython2 --jobs=auto \
```

Comment:
Even PHY targets are not the part of CI at the moment, please make the same changes for phy/broadcom/BCM81724/